### DISCUSSION TITLE
Keyboard shortcut for highlight text

### DISCUSSION THREAD

**Discussion by jrcacd** [standardnotes/forum#3429] [Suggestions]

**Comment by munkmode** [comment]
**+1, I would like to see this feature also.**

Kinda mindblowing that SN doesn't have keyboard shortcuts for all formatting options.

**Comment by dedfft** [comment]
Looks like this option is missing, and it would be really useful, especially for highlighting in super notes. Without it, manual mouse clicks are crazy(

**Comment by flourfie** [comment]
I cannot express my disappointment enough.
Use Super Note Type : Cannot add code block of any kind without using mouse (this is crazy in 2025).
Use Markdown to add code block with backticks at least: Cannot link other note in text.

I am only using Standard Notes because it's running on MacOs, Windows and ios, which is nice. But man, take a look at the Bear app editor, please. Or simply, I mean, Command+B makes text bold, Command+I makes in italic, please provide some shortcuts for code block and inline code and please, write them down somewhere in the documentation.
